app.pianocheetah.pianocheetah.json

```diff
@@ -0,0 +1,31 @@
+{
+   "app-id":  "app.pianocheetah.pianocheetah",
+   "command": "pianocheetah",
+   "sdk":     "org.kde.Sdk",
+   "runtime": "org.kde.Platform",
+   "runtime-version": "6.6",
+   "modules": [
+      {
+         "name": "cp_to_app",
+         "buildsystem": "simple",
+         "build-commands": [
+            "cp -pr share /app",
+            "cp -pr bin   /app"
+         ],
+         "sources": [
+            {
+               "type": "dir",
+               "path": "_build/files"
+            }
+         ]
+      }
+   ],
+   "finish-args": [
+      "--filesystem=host",
+      "--share=ipc",
+      "--share=network",
+      "--socket=wayland",
+      "--socket=fallback-x11",
+      "--socket=pulseaudio"
```

Comment:
This should be on top before modules.

Reply:
oh, that where I have it in my build from source manifest for flatpak-build.  I saw other manifests where it's like this.  It definitely works at this location.  Shall I change it in my branch and recommit/push ?  I've tested it.

Reply:
two wrong don't make a right. 

it should be before modules.

Reply:
ok fixed.